projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts

```diff
@@ -24,4 +24,145 @@ describe('GoButtonComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('clicked()', () => {
+    it('sets isProcessing to false if useLoader is false', () => {
+      expect(component.isProcessing).toBe(false);
+      component.clicked();
+      expect(component.isProcessing).toBe(false);
+    });
+
+    it('sets isProcessing to true if useLoader is true', () => {
```

Comment:
We could probably simplify this to just test that `isProcessing === useLoader` right?

Reply:
I see you updated the assertions in this test to reflect `isProcessing === useLoader`, but should we update the description to reflect this updated test?